### ISSUE TITLE
Mod Suggestions 

### ISSUE THREAD

**Issue by whatthedrunk** [AllTheMods/Remix#1]
Post them here! If it's a mod I dont care to add but you really want it let me know and maybe you can balance it for the pack.

**Comment by whatthedrunk** [comment]
All have been added to my local profile. I will have an update on the server tomorrow.

**Comment by acanady** [comment]
Project Red [base](https://minecraft.curseforge.com/projects/project-red-base), [Integration](https://minecraft.curseforge.com/projects/project-red-integration), [Fabrication](https://minecraft.curseforge.com/projects/project-red-fabrication), [Mechanical](https://minecraft.curseforge.com/projects/project-red-mechanical), [Lighting](https://minecraft.curseforge.com/projects/project-red-lighting), [World](https://minecraft.curseforge.com/projects/project-red-world), and [Compat](https://minecraft.curseforge.com/projects/project-red-compat). [Minecoprocessors](https://minecraft.curseforge.com/projects/minecoprocessors)
 [redstone paste](https://minecraft.curseforge.com/projects/redstone-paste)  [primitive mobs](https://minecraft.curseforge.com/projects/primitive-mobs)

**Comment by whatthedrunk** [comment]
This thread is for the unreleased remix pack we're working on.

**Comment by acanady** [comment]
yeah i edited my comment once i realized that, I got confused with this and the ATM3 repository :) Still, those mods would be cool in remix, depending on what you're trying to accomplish with the pack. Is it like a progression type pack?

**Comment by Blueletric** [comment]
This pack is based off the core 1.7 mods such as gregtech, thaumcraft, buildcraft and forestry. There are others but it is going to be one of the smaller ATM packs _so ive heard_

**Comment by Remyrios** [comment]
Some QoL mods:
- [Better Advancements](https://minecraft.curseforge.com/projects/better-advancements)
- [Nature's Compass](https://minecraft.curseforge.com/projects/natures-compass)
- [SimpleHarvest](https://minecraft.curseforge.com/projects/simpleharvest)
- [Spartan Shields](https://minecraft.curseforge.com/projects/spartan-shields)
- [Fullscreen Windowed](https://minecraft.curseforge.com/projects/fullscreen-windowed-borderless-for-minecraft)

and [Caliper](https://minecraft.curseforge.com/projects/caliper) as a really helpful modpack tool
